harvard-cranfield-university.csl

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" name-as-sort-order="all" demote-non-dropping-particle="sort-only" default-locale="en-GB">
   <info>
     <title>Harvard - Cranfield University</title>
     <title-short>Cranfield Harvard</title-short>
     <id>http://www.zotero.org/styles/harvard-cranfield-university</id>
     <link href="http://www.zotero.org/styles/harvard-cranfield-university" rel="self"/>
     <link href="http://www.zotero.org/styles/harvard-staffordshire-university" rel="template"/>
-    <link href="http://www.som.cranfield.ac.uk/som/dinamic-content/media/mirc/citing%20references%20for%20print%20A3.pdf" rel="documentation"/>
+    <link href="https://intranet.cranfield.ac.uk/Library/Cranfield/Documents/Author-date%20referencing%20guide.pdf" rel="documentation"/>
```

Comment:
if you have any authority over this, we'd much prefer a publicly available document.